### ISSUE TITLE
Neovim cursor random out of position

### ISSUE THREAD

**Issue by meijieru** [wez/wezterm#1639]
### What Operating System(s) are you seeing this problem on?

Windows

### WezTerm version

wezterm 20220101-133340-7edc5b5a

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

No

### Describe the bug

Mess up the content after `:PackerSync`. The cursor would be out of position randomly.
Contents are also messed up.

![image](https://user-images.githubusercontent.com/9511136/153737852-ab620a08-91a5-4855-a6db-e7a1e36f4dfd.png)

### To Reproduce

neovim version, actually happens no matter what version it is.
`NVIM v0.7.0-dev+1048-gdba1df6359`

### Configuration

no config

**Comment by wez** [comment]
Please try the nightly build

**Comment by meijieru** [comment]
Currently, no package updated lol.
Will try it again in several days.

**Comment by wez** [comment]
You can download the nightly from https://wezfurlong.org/wezterm/install/windows.html

**Comment by meijieru** [comment]
Sorry I mean neovim plugins lol.

**Comment by gegoune** [comment]
Is that by any chance related to cursor randomly changing positions in neovim on changing tabs/going out and back to Wezterm's window? I have seen something like that on occasion, not sure if it is the same issue as described here or if you are aware of it already. I did not manage to replicate it to open new issue.

**Comment by meijieru** [comment]
Confirm it exists in the nightly release.

**Comment by wez** [comment]
I need more information; I don't use neovim and I can't replicate your configuration.  I don't know what you are actually doing or what it is supposed to look like.

What usually helps in a situation like this is to use the unix `script` utility to record your terminal screen in a way that I can then replay the escape sequences--simply recording a video of the screen won't help me recreate the issue.

Please: 
* Set your terminal to 80x24 before starting recording, or if that size is too small, set it to the smallest reasonable size, and report that size here!  The size is important for replaying the script.
* Try to keep the recording short and focused to the thing that demonstrates the problem
* Add comments to this issue that help to pinpoint when it misbehaves so that I know what I'm looking for
